tests/MonitoringFunctions.Test/TestHelperMethods.cs

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+using Microsoft.Extensions.Logging.Abstractions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using MonitoringFunctions.Providers;
+using System.Threading.Tasks;
+
+namespace MonitoringFunctions.Test
+{
+    [TestClass]
+    public class TestHelperMethods
+    {
+        [TestMethod]
+        public async Task TestCheckUrlAccess()
+        {
+            using IDataService dataService = new DummyDataService();
+            await HelperMethods.CheckAndReportUrlAccess(NullLogger.Instance, "test_run", "https://www.microsoft.com", dataService);
+        }
```

Comment:
what is being tested?

Reply:
This tests that our method can successfully make an HTTP request and get result 200.

Reply:
my question relates to the absence of assertions in the test case method.  The scheme is: if the method completes, the method passes, without verifications.

Reply:
`Assert.ThrowsException` doesn't seem applicable here. Method assumes that the test will succeed when there is an exception of given type, whereas in our case an exception means failure. As we discussed, instead of adding Assert for the sake of Assert, I added a comment with hopes of clarifying the situation a bit.